jvector-examples/src/main/java/io/github/jbellis/jvector/example/Bench.java

```diff
@@ -71,7 +71,7 @@ public static void main(String[] args) throws IOException {
         );
         List<EnumSet<FeatureId>> featureSets = Arrays.asList(
                 EnumSet.of(FeatureId.NVQ_VECTORS),
-//                EnumSet.of(FeatureId.NVQ_VECTORS, FeatureId.FUSED_ADC),
+//                EnumSet.of(FeatureId.NVQ_VECTORS, FeatureId.FUSED_PQ),
                 EnumSet.of(FeatureId.INLINE_VECTORS)
```

Comment:
Do we want to uncomment this now that the feature is implemented?

Reply:
We should in reality deprecate Bench altogether. BenchYAML has become a much more handy point of entry. But I agree, I will uncomment this.

Reply:
Done!